package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Bot for handling custom behaviors on Twitch stream",
   "main": "bot.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "set NODE_ENV=prod&& node bot.js"
```

Comment:
does this actually work? Does `SET` in bash put the variable into env? Wouldn't you need `export`?

Reply:
Worked for older setup when config was setting twitch settings. really just used in development to npm start the bot. deprecated